mache/spack/pm-cpu_intel_mpich.sh

```diff
@@ -50,4 +54,8 @@ export FI_CXI_RX_MATCH_MODE=software
 if [ -z "${NERSC_HOST:-}" ]; then
   # happens when building spack environment
   export NERSC_HOST="perlmutter"
+export FI_CXI_RX_MATCH_MODE=software
+export MPICH_COLL_SYNC=MPI_Bcast
+export GATOR_INITIAL_MB=4000MB
+export BLA_VENDOR=Intel10_64_dyn
```

Comment:
These also need to be moved outside of the `if`/`fi`.

Reply:
whoops! Just fixed these two. Going to re-run the intel test now.

Reply:
I did already.  It unfortunately didn't help.